### ISSUE TITLE
Failed testcases Temporal Analogy

### ISSUE THREAD

**Issue by l591326240** [opennars/opennars#32]
```
<(&/,<(*,Robot_01,Gear) --> D>,<(*,Robot_01,Accelerator) --> Step_on>) 
<|> 
(&/,<(*,Robot_01,feet) --> D>,<(*,Robot_01,Car_02) --> Step_on>)>. 

<(&/,<(*,Robot_01,Car_02) --> Turn_on>,<(*,Robot_01,Hand_Brake) --> Lift>) 
=\> 
(&/,<(*,Robot_01,Gear) --> D>,<(*,Robot_01,Accelerator) --> Step_on>)>.

Unexpected output: <(&/, <(*, Robot_01, Gear) --> D>, <(*, Robot_01, Accelerator) --> Step_on>) <|> (&/, <Gear --> (/, D, Robot_01,_)>, <(*, Robot_01, Accelerator --> Step_on>)> %1.00, 0.81%

--------------------------------------------------------------------------------------------------------

```

**Comment by patham9** [comment]
Can you bring them in correct form? The above statements are not valid NAL so it is hard to say if you misused Narsese or if those are real bugs. 

Best regards, 
Patrick

**Comment by l591326240** [comment]
Just sent you an email

**Comment by patham9** [comment]
Fixed in a93620c5c01143ecffbbba4802249dd4f215073b commit.

**Comment by patham9** [comment]
Hm this is not the issue I fixed, why didn't you make a new Issue Entry instead of removing the examples this issue was originally about?

Best regards,
Patrick

**Comment by patham9** [comment]
For documenting, this issue was originally about the order issue:
```
<a <|> b>.
<b =/> c>.
|-
<a =\> c>.
where it should have been
<a =/> c>.
```

**Comment by patham9** [comment]
The new issue is fixed in e1ccaf5e555611bd644648af2c8d979a6689de43
